docs/devguide/opendds_idl.rst

```diff
@@ -313,9 +316,15 @@ Unlike when using the classic mapping, ``Foo.idl`` is not processed by ``tao_idl
 
   * Note that setting a union value through a modifier method automatically sets the discriminator.
     In cases where there are multiple possible values for the discriminator, a 2-argument modifier method is provided.
-    Using this is preferred to using _d().
+    Using this is preferred to using ``_d()``.
+
+  * .. compound::
+
+      If you chose to use the ``_d()`` method of the generated union types, note the following requirement from the specification:
+
+        The ``_d`` discriminator modifier can only be used to set the discriminant to a value within the same union member.
```

Comment:
``` suggestion
        The ``_d`` discriminator modifier can only be used to set the discriminant to a value that has the same union member as currently selected.
```

Reply:
That would probably be a better way of phrasing it, but this is quoting the spec. I will add the section number it's quoting because it's missing, but I don't think the quote should be modified as long it's in the latest published spec.

Reply:
I don't think it has to directly quote the spec (but of course we'd need to reword the paragraph above) if there's a better way of explaining it in the DevGuide.

Reply:
I have created an OMG issue on the spec